WooCommerce/Classes/ViewRelated/Products/Variations/Edit Attributes/AttributePickerViewController.swift

```diff
@@ -170,6 +188,12 @@ private extension AttributePickerViewController {
         tableView.reloadData()
         updateDoneButton()
     }
+
+    private func presentBackNavigationActionSheet() {
```

Comment:
nit: extension is already private
```suggestion
    func presentBackNavigationActionSheet() {
```